addons/account/models/account_move.py

```diff
@@ -80,7 +80,7 @@ class AccountMove(models.Model):
     _name = "account.move"
     _inherit = ['portal.mixin', 'mail.thread.main.attachment', 'mail.activity.mixin', 'sequence.mixin', 'product.catalog.mixin']
     _description = "Journal Entry"
-    _order = 'date desc, name desc, invoice_date desc, id desc'
+    _order = 'date desc, name desc, invoice_date desc, id'
```

Comment:
Why does the order change here? The new order seems counter intuitive since all the other order clauses are to have the most recent first.

Reply:
If a move neither has a `date`, nor a `name`, nor an `invoice_date`, the document has nothing that makes it the most recent. I think when that's the case, we should keep them in the order of creation so that some matching process that create draft entries then match them keep a relevant order. Not matching first created with second source and second created with first source for example.